generators/app/templates/src/app/screens/Login/layout.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import i18next from 'i18next';
+
+import InputLabel from '../../components/InputLabel';
```

Comment:
@components?

Reply:
We are temporarily removing babel module resolver as it broke with babel 7. It'll return very soon in a new PR